conformance/utils/suite/suite.go

```diff
@@ -74,8 +74,28 @@ const (
 
 // StandardCoreFeatures are the features that are required to be conformant with
 // the Core API features that are part of the Standard release channel.
-var StandardCoreFeatures = map[SupportedFeature]bool{
-	SupportReferenceGrant: true,
+var StandardCoreFeatures = sets.Set[SupportedFeature]{
+	SupportReferenceGrant: {},
+}
+
+// AllFeatures contains all the supported features and can be used to run all
+// conformance tests with `all-features` flag.
+//
+// Note that the AllFeatures must in sync with defined features when the
+// feature constants change.
+var AllFeatures = sets.Set[SupportedFeature]{
+	SupportReferenceGrant:                     {},
+	SupportTLSRoute:                           {},
+	SupportHTTPRouteQueryParamMatching:        {},
+	SupportHTTPRouteMethodMatching:            {},
+	SupportHTTPResponseHeaderModification:     {},
+	SupportRouteDestinationPortMatching:       {},
+	SupportGatewayClassObservedGenerationBump: {},
+	SupportHTTPRoutePortRedirect:              {},
+	SupportHTTPRouteSchemeRedirect:            {},
+	SupportHTTPRoutePathRedirect:              {},
+	SupportHTTPRouteHostRewrite:               {},
+	SupportHTTPRoutePathRewrite:               {},
 }
```

Comment:
It should be possible to define these this way, right?
```suggestion
var StandardCoreFeatures = sets.New(
	SupportReferenceGrant,
)

// AllFeatures contains all the supported features and can be used to run all
// conformance tests with `all-features` flag.
//
// Note that the AllFeatures must in sync with defined features when the
// feature constants change.
var AllFeatures = sets.New(
	SupportReferenceGrant,
	SupportTLSRoute,
	SupportHTTPRouteQueryParamMatching,
	SupportHTTPRouteMethodMatching,
	SupportHTTPResponseHeaderModification,
	SupportRouteDestinationPortMatching,
	SupportGatewayClassObservedGenerationBump,
	SupportHTTPRoutePortRedirect,
	SupportHTTPRouteSchemeRedirect,
	SupportHTTPRoutePathRedirect,
	SupportHTTPRouteHostRewrite,
	SupportHTTPRoutePathRewrite,
}
```

Reply:
@shaneutt Yes, I've changed like this. Thanks.